docs/marketplace/add_to_marketplace/add_item_by_type/add_template_or_example.md

```diff
@@ -0,0 +1,225 @@
+
+Check out the [Node.js service template](https://github.com/mia-platform-marketplace/Node.js-Custom-Plugin-Template/blob/255233ce35ec7748bb4120057dc36fcd2bb3f983/Dockerfile#L29-L30) to see an example.
+
+## The source Git repository
+
+
+The item template must be hosted on a remote **Git repository**.
+
+ The URL to the tar.gz version of the Git project must be specified in the `archiveUrl` field, optionally you can specify either the `sha` or the `ref` as query parameters. If you don't have access to CMS, this should be provided to your Mia-Platform referent in form of a Git clone URL.
+
+The service should ensure a Continuous Integration (CI) to update the service image any time his code is modified, through the `pipelines` field you can set up the CI file by yourself or use instead Mia-Platform pipeline templates.  
+
+To use the templates provided by Mia-Platform, you can contact your referent to configure the pipeline templates within your Console installation. 
+
+When creating your microservice, the Console will also generate a CI file (e.g. `.gitlab-ci.yml`) alongside the service files.
+- The service must be well documented and the field `documentation` helps in it.
+
+In fact, during service creation, it is possible to access the service documentation by clicking on `View documentation` button, which will appear only if the `documentation` field has been filled correctly.  
+Two properties must be specified inside `documentation`:  
+
+- `type`, currently only two types exist:
+- `markdown`: represents a markdown file (with `.md` file extension), for example, a `README.md` file inside a Git repository.
+- `externalLink`: represents a link to an external website page, for example to Mia Platform documentation.
+- `url`, contains the URL where the markdown file can be retrieved (if its type is `markdown`), or the link where the user should be redirected (if its of type `externalLink`).  
+
+:::info
+
+By adding an **Example** or a **Template**, the code provided will act as a boilerplate for everyone creating a service from that Marketplace item, therefore that code has to be accessible to all the Console users that have access to the Marketplace item.
+
+:::  
+
+Once the user creates a new microservice starting from your template or example, the Console will create a new Git repository in which it will copy all the template files.
+
+:::info
+
+Further details on creating a template are provided by visiting [this documentation page](/marketplace/templates/template_create.md).
+
+:::
+
+## Repository Files Placeholder
+
+The files of your template repository can contain some special placeholders that will be replaced by the Console at the creation of the new microservice:
+
+- `mia_template_image_name_placeholder`: name of the docker image entered by the user;
+- `%CUSTOM_PLUGIN_PROJECT_NAME%`: name (label) of the Console project;
+- `mia_template_project_id_placeholder`: id of the Console project;
```

Comment:
perchè questi non hanno la stessa sintassi?

Reply:
perché spaccherebbero la sintassi del Dockerfile o del file in cui si trovano

Reply:
ad esempio qui spaccherebbe il `go.mod` https://github.com/mia-platform-marketplace/Go-Template/blob/3205405043b56e6d36a7ed36c7d65eef6b82fa8a/go.mod#L1